<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>564</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>4</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeWidget" name="treeWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>151</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTreeView::item {
    border-bottom: 1px solid #d6d6d6; /* Add a bottom border to each item */
    padding: 5px; /* Add padding to make item taller */
	 background-color: rgb(240, 240, 240)

    
}
QTreeWidget {
    background-color: rgb(240, 240, 240); /* 使用实际的背景色值 */
}</string>
    </property>
    <column>
     <property name="text">
      <string>客服质检</string>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
    </column>
    <item>
     <property name="text">
      <string>概览</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>质检规则和方案</string>
     </property>
     <property name="foreground">
      <brush brushstyle="Dense4Pattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
     <item>
      <property name="text">
       <string>质检规则配置</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>质检方案管理</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>质检任务</string>
     </property>
     <item>
      <property name="text">
       <string>任务管理</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>数据集管理</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>待办任务</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>系统管理</string>
     </property>
    </item>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>671</width>
      <height>521</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="whatsThis">
     <string comment="d"/>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="currentIndex">
     <number>9</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>概览</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>质检规则配置</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QTableView" name="RuleManagerTableView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>671</width>
        <height>461</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
     <widget class="QPushButton" name="AddRuleButton">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>20</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>+ 新建规则</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>质检方案管理</string>
      </property>
     </widget>
     <widget class="QTableView" name="scheme_tableView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>671</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="new_scheme_pushButton">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>10</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>新建方案</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>任务管理</string>
      </property>
     </widget>
     <widget class="QTableView" name="task_tableView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>671</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="new_task_pushButton">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>10</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>+新质检任务</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>数据集管理</string>
      </property>
     </widget>
     <widget class="QTableView" name="DataSetManagerTableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>671</width>
        <height>461</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
     <widget class="QPushButton" name="new_dataset_pushButton">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>10</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>+ 导入数据集</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>待办任务</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_7">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>系统管理</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_8">
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>规则编辑</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea_rule_edit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>661</width>
        <height>401</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>659</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>规则名称：</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="RuleNameEditText">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="save_rule_pushButton">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
     <widget class="QPushButton" name="NewRuleButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>+ 新条件</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_rule_pushButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
     <widget class="QComboBox" name="score_type_comboBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>一次打分为</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>加减总得分</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_score_standard">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>评分规则：</string>
      </property>
     </widget>
     <widget class="QLabel" name="score_value_Label">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>70</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>分</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="score_value_line_edit">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>70</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_9">
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>方案编辑页面</string>
      </property>
     </widget>
     <widget class="QPushButton" name="save_scheme_pushButton">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>20</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>保存方案</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_scheme_pushButton">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>20</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>删除方案</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>已包含规则：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>方案名称：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="scheme_name_lineEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="existing_rules_in_scheme_tableView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>421</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="add_rule_to_scheme_pushButton">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>140</y>
        <width>21</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_rule_from_scheme_pushButton">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>140</y>
        <width>21</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>方案描述：</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="scheme_description_textEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_10">
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>新建质检任务</string>
      </property>
     </widget>
     <widget class="QTableView" name="choose_dataset_tableView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>641</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="pick_dataset_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>选择数据集</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>60</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="text">
       <string>→</string>
      </property>
     </widget>
     <widget class="QLabel" name="pick_scheme_label_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>40</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>选择方案</string>
      </property>
     </widget>
     <widget class="QLabel" name="start_analyze_label">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>40</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>开始质检</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>60</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="text">
       <string>→</string>
      </property>
     </widget>
     <widget class="QPushButton" name="next_step_pushButton">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>350</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>下一步</string>
      </property>
     </widget>
     <widget class="QPushButton" name="previous_step_pushButton">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>350</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>上一步</string>
      </property>
     </widget>
     <widget class="QLabel" name="task_name_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>质检任务名称*：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="task_name_lineEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>411</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="task_description_textEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>160</y>
        <width>411</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="task_description_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>质检任务描述*：</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="manual_check_checkBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>230</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>需要人工复检</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>scrollArea_rule_edit</tabstop>
  <tabstop>treeWidget_3</tabstop>
  <tabstop>RuleManagerTableView</tabstop>
  <tabstop>NewRuleButton</tabstop>
  <tabstop>save_rule_pushButton</tabstop>
  <tabstop>RuleNameEditText</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
